docs/General/pagination.md

```diff
@@ -0,0 +1,25 @@
+--8<-- "References/abbreviations.md"
+
+1. Go back to the Sample Web UI on your development system.
+
+2. click any tab (wireless, cira configs, domains, profiles and devices) from the menu on the left.
```

Comment:
Is there a reason you switched to lower case at the beginning of each step?

Click any tab (wireless, cira configs, domains, profiles and devices) from the menu on the left.